app/views/mailers/pipeline_mailer/complete_email.html.erb

```diff
@@ -0,0 +1,93 @@
+<div
+  role="article"
+  aria-roledescription="email"
+  aria-label="Confirm your email address"
+  lang="en"
+>
+  <div
+    style="background-color: #f8fafc; font-family: ui-sans-serif, system-ui, -apple-system, 'Segoe UI', sans-serif"
+  >
+    <table align="center" cellpadding="0" cellspacing="0" role="none">
+      <tr>
+        <td class="w-full" style="width: 552px;">
+          <div class="text-center my-12">
+            <a
+              href="<%= root_url %>"
+              class="whitespace-nowrap"
+              style="text-decoration: none;"
+            >
+              <span class="font-semibold text-black text-5xl">
+                <%= t(:"general.name") %>
+              </span>
+            </a>
+          </div>
+          <table class="w-full" cellpadding="0" cellspacing="0" role="none">
+            <tr>
+              <td
+                class="py-12 px-6 rounded bg-white"
+                style="font-size: 16px; color: #334155; box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.05)"
+              >
+                <h1 class="mt-0 mx-0 mb-6 font-semibold text-black" style="font-size: 24px;">
+                  <%= t(:".greeting", name: @workflow_execution.submitter.first_name) %>&#44;
+                </h1>
+                <p class="m-0" style="line-height: 24px">
+                  <%= t(:".body.start") %>
+                  <strong><%= @workflow_execution.id %></strong>
+                  <%= t(:".body.end") %>
+                  <br>
+                  <br>
+                  <%= t(:".view_details") %>
+                </p>
+                <div role="separator" style="line-height: 24px">&zwj;</div>
+                <div>
+                  <a
+                    href="<%= workflow_execution_url(@workflow_execution) %>"
+                    class="
+                      inline-block
+                      rounded
+                      py-4
+                      px-6
+                      font-semibold
+                      leading-none
+                    "
+                    style="background-color: #15803d; font-size: 16px; color: #f8fafc; text-decoration: none"
+                  >
+                    <span style="mso-text-raise: 16px">
+                      <%= t(:"general.name") %>
+                      &rarr;
+                    </span>
+                  </a>
+                  <div role="separator" style="line-height: 24px">&zwj;</div>
+                  <p class="m-0" style="line-height: 24px">
+                    <%= t(:".copy_paste") %>
+                    <a href="<%= workflow_execution_url(@workflow_execution) %>">
+                      <%= workflow_execution_url(@workflow_execution) %>
```

Comment:
I think having this underlined since it's a link would be good.

Reply:
2f949044248a246e6e9240fc5d17373d9f1f0881